lighthouse-core/test/audits/seo/font-size-test.js

```diff
@@ -137,6 +137,111 @@ describe('SEO: Font size audit', () => {
     expect(auditResult.displayValue).toBeDisplayString('0% legible text');
   });
 
+  it('attributes source location', async () => {
+    // From external stylesheet.
```

Comment:
can we make each of these separate test cases? :)

Reply:
done